Code/Source/Lidar/LidarTemplateUtils.cpp

```diff
@@ -74,8 +75,8 @@ namespace ROS2
             for (int layer = 0; layer < lidarTemplate.m_layers; layer++)
             {
                 // roll is equal to 0, so it's skipped in the calculations
```

Comment:
we should not leave out roll transformation

Reply:
Agreed, but not the scope of this PR. Most probably we will move towards quaternions anyway.